### DISCUSSION TITLE
Custom module > Global Search

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#17147] [How to]
Users (not admin) cannot search records in Custom module with use of Global Search. I am not sure if maybe something is misconfigured. Can I have a tips on that?

Best!

**Comment by rskrzypczak** [comment]
Permissions.
Users may not have permission to these records, and if they do, they probably have to wait for cron to recalculate them.
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/9f8f738c-3d23-44f5-bd75-608e58001f8d)

**Reply by garryatbrighton** [reply]
Seems like it is not that. Users have read only access. Looking at profile Privileges to actions and views I don't see logically connected to this case checkbox, or it might be depended on one? Cron running OK, still users cannot see global search results

**Reply by Akaki-1991** [reply]
did you enable "track changes"?

**Comment by mariuszkrzaczkowski** [comment]
check your module here
https://gitdeveloper.yetiforce.com/index.php?module=Search&parent=Settings&view=Index

**Comment by ArniSamano** [comment]
Noticed the same with my custom module. Checked (probably) all related configs, and somehow one of my basic custom module cannot be used in global search. Help appreciated

Cheers!
